geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/RedisListTest.java

```diff
@@ -173,4 +195,10 @@ private RedisList createRedisList(int e1, int e2) {
     newList.elementPushHead(new byte[] {(byte) e2});
     return newList;
   }
+
+  private RedisList createRedisListWithOneElement(int e1) {
+    RedisList newList = new RedisList();
+    newList.elementPushHead(new byte[] {(byte) e1});
```

Comment:
For simplicity, could this method just take a `byte[]` as the argument rather than an int that we then have to fiddle with to put into the list?

Reply:
Since we don't care about the contents of the element, now there's no arg at all.